### ISSUE TITLE
[Bug] prefer-to-have-count false positive

### ISSUE THREAD

**Issue by CreativeTechGuy** [playwright-community/eslint-plugin-playwright#207]
Currently `prefer-to-have-count` checks for 3 things within the expect call:

* an await expression
* a function call
* a member expression

This rule should be specifically checking for the method `.count()` like the name describes. Currently it picks up other things like the following:

```js
expect(await page.evaluate(() => {
    return document.querySelectorAll("*").length;
})).toBe(10);
```

That sample above is incorrectly reported as an error since it matches all of the 3 conditions despite it not being a `.toHaveCount()` compatible call.

Thank you! :)
